lua/lv-which-key/init.lua

```diff
@@ -179,17 +134,42 @@ local mappings = {
         s = {"<cmd>SessionSave<cr>", "Save Session"},
         l = {"<cmd>SessionLoad<cr>", "Load Session"}
     },
-    -- extras
-    z = {
-        name = "Zen",
-        s = {"<cmd>TZBottom<cr>", "toggle status line"},
-        t = {"<cmd>TZTop<cr>", "toggle tab bar"},
-        z = {"<cmd>TZAtaraxis<cr>", "toggle zen"}
-    }
+}
+
+local visualMappings = {
```

Comment:
Please add a note above this which says:
-- NOTE: Prefer using : over \<cmd> in the command expression as the later avoids going back in normal-mode.
-- see https://neovim.io/doc/user/map.html#:map-cmd